src/utility/Validation.js

```diff
@@ -172,6 +170,7 @@ const validators = {
     resume: noInvalidResumeFunction,
     portfolio: isDesigner ? mandatoryURLFunction : optionalURLFunction,
     linkedin: optionalURLFunction,
+    hackathonsAttended: noEmptyFunction,
```

Comment:
Why does moving this under the `skills` field fix this? Just want to check because the current template still has `hackathonsAttended` under `basicInfo` in `Constants.js`

Reply:
Resolved through Slack, `hackathonsAttended` was moved to Skills section